examples/mnist.py

```diff
@@ -66,7 +72,11 @@ def load_data():
 
 def build_model(input_dim, output_dim,
                 batch_size=BATCH_SIZE, num_hidden_units=NUM_HIDDEN_UNITS):
-
+    """Create a symbolic representation of a neural network with
+    `intput_dim` input nodes, `output_dim` output nodes, a training function
+    with a mini-batch size of `batch_size` and `num_hidden_units` per hidden
+    layer.
```

Comment:
I've updated the docstring.

Isn't it the case that the training function has to have a batch size of `batch_size`? If not, why do we pass this parameter? What would happen if we passed `batch_size=8` to `build_model` and `batch_size=9`  to `train`?